w3-account.md

```diff
@@ -12,61 +12,93 @@
+Agent authorization can use familiar email-based authorization flows providing a smooth onboarding experience and hide complexity of the underlying [PKI]. This approach also better reflects user intuition: they have ambient authority over owned spaces and can authorize user agents (think apps) giving them necessary level of access.
 
-Using an account identifier based on a memorable email address solves the discovery problem, and email-based authorization flows provide a smoother onboarding experience by hiding the complexity of [PKI]. With this approach, users can aggregate all of their delegations under a single account identifier and re-delegate desired capabilities to other agents.
+> ℹ️ This specification mainly focuses on [`did:mailto`] identifiers, but implementations are free to extend it to various other [DID methods].
+>
+> Also, note that [account] is defined as non [`did:key`] identifier because [`did:key`] identifiers can aggregate and re-delegate capabilities natively with [UCAN]s. [Account]s simply bring human-meaningful identifiers that users can type in on every agent. Use of out of bound authorization also frees users from key management.
 
-> ℹ️ This specification mainly focuses on [`did:mailto`] identifiers, but it can be extended to various other types of identifiers.
+## Serialization
 
-# High-Level Concepts
+[UCAN]s MUST be encoded with some [IPLD] codec. [DAG-CBOR] is RECOMMENDED.
+
+## Concepts
 
 ## Roles
 
 There are several distinct roles that [principals] may assume in described specification:
 
 | Name        | Description                                                                                                                                    |
 | ----------- | ---------------------------------------------------------------------------------------------------------------------------------------------- |
+| Principal | The general class of entities that interact with a UCAN. Listed in the `iss` or `aud` field |
 | Account    | [Principal] identified by memorable identifier like [`did:mailto`]. |
 | Agent       | [Principal] identified by [`did:key`] identifier, representing a user in some application installation |
+| Issuer | Principal sharing access. It is the signer of the [UCAN]. Listed in the `iss` field |
+| Audience | Principal access is shared with. Listed in the `aud` field |
+
+### Space
+
+Namespace, or space for short, is an owned resource that can be shared. It corresponds to the asymmetric keypair and is identified by the [`did:key`] URI.
+
+Space is always listed in the `with` field of the [UCAN] capability.
+
+### Owner
+
+The [owner] of the [space] is the holder of its private key. Space owner can share limited or full access to owned space via [UCAN] delegation issued by the [space] [`did:key`] and signed with a [space] private key.
```

Comment:
can we call it [controller](https://www.w3.org/TR/did-core/#dfn-did-controllers) vs owner?

Reply:
Sounds like it would be more confusing than helpful. According to linked definition controller is entity that is able to modify DID document, but in case of `did:key` no one actually can modify did document even private key holder.

I'm amendable to this, but personally owner of the space seems more intuitive to me and I do not consider it to be the same thing as DID controller.